webrtc/RTCRtpParameters-headerExtensions.html

```diff
@@ -63,13 +63,44 @@
 
     param.headerExtensions = [{
       uri: 'non-existent.example.org',
+      id: 404
+    }];
+
+    return promise_rejects(t, 'InvalidModificationError',
+      sender.setParameters(param));
+
+  }, `setParameters() with modified headerExtensions should reject with InvalidModificationError, checks that encrypted is declared false`);
+
+  promise_test(t => {
+    const pc = new RTCPeerConnection();
+    const { sender } = pc.addTransceiver('audio');
+    const param = sender.getParameters();
+    validateSenderRtpParameters(param);
+
+    param.headerExtensions = [{
+      uri: 'non-existent.example.org',
+      encrypted: false
+    }];
+
+    return promise_rejects(t, 'InvalidModificationError',
+      sender.setParameters(param));
+
+  }, `setParameters() with modified headerExtensions should reject with InvalidModificationError, checks that id is a required field`);
```

Comment:
This test doesn't test anything at all. You can't add new header extensions, and the error code (InvalidModificationError) is the same for all cases.